docs/projects/ad9083_evb/index.rst

```diff
@@ -0,0 +1,316 @@
+   * - PS
+     - SPI 0
+     - AD9083
+     - 0
+   * - PS
+     - SPI 0
+     - AD9528
+     - 1
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 20 20 20 15
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+     - Zynq MP
+   * - refsel
+     - INOUT
+     - 34
+     - 181:150
+     - 205:174
+   * - rstb
+     - INOUT
+     - 33
+     - 149:118
+     - 173:142
+   * - pwdn
+     - INOUT
+     - 32
+     - 86
+     - 110
+
+.. note::
+
+   For A10SoC project, the GPIO numbers coincide with the HDL GPIO EMIO number
+   (column 3 from the table).
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+================== === ============ ============= ======
+Instance name      HDL Linux ZynqMP Actual ZynqMP A10SoC
+================== === ============ ============= ======
+axi_ad9083_rx_dma  13  109          141           ---   
+axi_ad9083_rx_jesd 12  108          140           ---   
+axi_ad9083_dma     12  ---          ---           31    
+ad9083_jesd204     11  ---          ---           32    
+================== === ============ ============= ======
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI distributes the bit/elf files of these projects as part of the
+:dokuwiki:`ADI Kuiper Linux <resources/tools-software/linux-software/kuiper-linux>`.
+If you want to build the sources, ADI makes them available on the
+:git-hdl:`HDL repository </>`. To get the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the hdl/projects/$eval_board/$carrier location and run the make
+command.
+
+**Linux/Cygwin/WSL**
+
+Building the project on ZCU102 without parameters, will use the default
+configuration, meaning L = 4, M = 16 and L = 1.
+
+.. shell::
+
+   $cd hdl/projects/ad9083_evb/zcu102
+   $make
+
+Example of running the ``make`` command with parameters:
+
+.. shell::
+
+   $cd hdl/projects/ad9083_evb/zcu102
+   $make RX_JESD_L=4 RX_JESD_M=16
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make RX_JESD_L=4 RX_JESD_M=16``
+
+then the folder name will be:
+
+``RXL4_RXM16``
+because of truncation of some keywords so the name will not exceed the limits
+of the Operating System (``JESD``, ``LANE``, etc. are removed) of 260
+characters.
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Software considerations
+-------------------------------------------------------------------------------
```

Comment:
Do we need to keep this if there are no Software considerations?

Reply:
done

Reply:
done